hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/api/filter/AuthenticationFilter.java

```diff
@@ -72,7 +72,8 @@ public class AuthenticationFilter implements ContainerRequestFilter {
 
     private static final List<String> WHITE_API_LIST = ImmutableList.of(
             "auth/login",
-            "versions"
+            "versions",
+            "openapi.json"
```

Comment:
we add swagger file to white_list by default to allow normal users to check the api & not locked by auth